### ISSUE TITLE
Bidirectional gateway part 2

### ISSUE THREAD

**Issue by aspiers** [nicolasbock/github-mailing-list#11]
This is a much longer-term dream / item for the wish-list, following on from #10:

Somehow make it so that when people give reviews via GitHub's normal PR web UI or API, the reviews automatically get sent back to the review email thread.  Of course GitHub already sends email notifications, but they suck:

- https://github.com/isaacs/github/issues/836
- They don't give any context from the discussion.

So you are forced to just follow the link to the original PR on the web UI and then find the bit you're interested in.

This is probably substantially harder than part 1 ...

Addressing this would achieve the second stage of a bidirectional gateway, so that people could independently choose whether to review via email or via the web UI / API, and everything would work together seamlessly :-)

**Comment by nicolasbock** [comment]
Doesn't GitHub have all these hooks? Isn't there one for new messages?

**Comment by aspiers** [comment]
Yeah, I guess you'd have to implement it as [a webhook](https://developer.github.com/webhooks/), but that would probably work fine.
